camunda-scaling-operator/config/rbac/role.yaml

```diff
@@ -4,6 +4,31 @@ kind: ClusterRole
 metadata:
   name: manager-role
 rules:
+- apiGroups:
```

Comment:
Same here. The kubebuilder marker for permissions in the controller file makes sure that it generates the correct k8s role for the operator.